scanner/src/main/kotlin/Main.kt

```diff
@@ -138,6 +138,11 @@ object Main {
             order = PARAMETER_ORDER_OPTIONAL)
     private var summaryFormats = listOf(OutputFormat.YAML)
 
+    @Parameter(description = "The path to executable of Askalono tool. Makes sense only with --scanner askalono",
```

Comment:
Missing dot at end of sentence, also please put '--scanner askalono' in single quotes.